### PR TITLE
Fix _extend fn - alter provided object

### PR DESCRIPTION
**Pull request by jakubhruby**
Fixes https://github.com/fullstack-build/tslog/issues/285

### CODE REVIEW

File: src/runtime/browser/util.inspect.polyfil.ts [hunk 1]
```diff
@@ -381,13 +381,12 @@ function _extend(origin: object, add: object) {
   // Don't do anything if add isn't an object
   if (!add || !isObject(add)) return origin;
 
-  const clonedOrigin = { ...origin } as { [key: string]: unknown };
```
Comment (jakubhruby) [inline src/runtime/browser/util.inspect.polyfil.ts:384]: This was not needed - it lived just in this method

File: src/runtime/browser/util.inspect.polyfil.ts [hunk 2]
```diff
@@ -381,13 +381,12 @@ function _extend(origin: object, add: object) {
...
   const clonedAdd = { ...add } as { [key: string]: unknown };
 
   const keys = Object.keys(add);
   let i = keys.length;
   while (i--) {
-    clonedOrigin[keys[i]] = clonedAdd[keys[i]];
+    origin[keys[i]] = clonedAdd[keys[i]];
   }
   return origin;
```
Comment (jakubhruby) [inline src/runtime/browser/util.inspect.polyfil.ts:391]: Passed object was returned with no change

**Comment by terehov** [comment]
Thank you. I have released an update with your changes 👍

**Comment by jakubhruby** [comment]
This later commit breaks it again https://github.com/fullstack-build/tslog/commits/v4.9.3/#diff-f37f417161a3f575bc97f812d0cc6c5da9a62f6fe5abbb342b4b2789ecad9ada

**Pull request merged**